docs/user/plagiarism-check.rst

```diff
@@ -172,6 +173,73 @@ The process is visualized in the following diagram.
 2. Reply to the message of the instructor.
 3. Wait for the final verdict.
 
+Continuous Plagiarism Control
+-----------------------------------------
+
+In this section, we explain the automated flow of the continuous plagiarism control (CPC) and present actions taken by different Artemis users.
+
+Automated Flow
+^^^^^^^^^^^^^^
+
+The CPC is an experimental feature designed to shorten the feedback loop for students and reduce the manual work required from tutors and instructors.
+Continuous feedback on potential plagiarism aims to encourage students to submit their original work and enhance their learning outcomes.
+The continuous plagiarism control **does not accuse students of plagiarism, nor does it affect their scores**.
+Instead, it **notifies students of significant similarities** detected between their submissions, allowing them to revise their solutions.
+
+The CPC initiates plagiarism checks every night for all onboarded exercises, processing all the latest submissions for each exercise.
+If a significant similarity is detected, the CPC creates or updates a significant similarity plagiarism case for that submission.
```

Comment:
What is a `significant similarity plagiarism case` and how does it relate to "normal" plagiarism cases?

Reply:
I added a sentence explaining what it is